py-polars/tests/unit/io/test_excel.py

```diff
@@ -26,7 +26,7 @@ def test_read_excel(excel_file_path: Path) -> None:
 
 
 def test_read_excel_all_sheets(excel_file_path: Path) -> None:
-    df = pl.read_excel(excel_file_path, sheet_id=None)  # type: ignore[call-overload]
+    df = pl.read_excel(excel_file_path, sheet_id=0)
```

Comment:
previously the test had `sheet_id=None`, but what was documented was to pass `0` to get all sheets